### PR TITLE
Instrument OpenTracing example service2 using the JavaAgent.

### PR DESCRIPTION
**Pull request by objectiser**
Fix issue with baggage propagating Camel prefixed headers, causing outbound component to throw exception.
Added agent rule test.

### CODE REVIEW

**Comment by objectiser** [comment]
@davsclaus Updated one of the services in the opentracing example so that it uses the new OpenTracing Java Agent to register the tracer - with the rule now being distributed in the camel-opentracing component. If you could review please?

File: components/camel-opentracing/src/main/resources/otagent/apache-camel.btm [hunk 1]
```diff
@@ -0,0 +1,16 @@
+
```
Comment (davsclaus) [inline components/camel-opentracing/src/main/resources/otagent/apache-camel.btm]: Can an ASF license header be added to this file?

File: components/camel-opentracing/src/test/java/org/apache/camel/opentracing/agent/InstallOpenTracingTracerRuleTest.java [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+import org.apache.camel.ProducerTemplate;
+import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.component.mock.MockEndpoint;
+import org.apache.camel.test.junit4.CamelTestSupport;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+/**
+ * @author gbrown
```
Comment (davsclaus) [inline components/camel-opentracing/src/test/java/org/apache/camel/opentracing/agent/InstallOpenTracingTracerRuleTest.java]: We dont have author tags in javadoc, would you kindly remove this

**Comment by objectiser** [comment]
@davsclaus updates applied.

**Comment by davsclaus** [comment]
Thanks for the PR it has been merged. Do you mind closing this?

**Comment by objectiser** [comment]
@davsclaus Thanks!

**Pull request closed without merging**